flowapi/app/geography.py

```diff
@@ -0,0 +1,81 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from quart import Blueprint, current_app, request, stream_with_context, jsonify
+from .check_claims import check_claims
+
+blueprint = Blueprint(__name__, __name__)
+
+
+@blueprint.route("/geography/<aggregation_unit>")
+@check_claims("get_result")
+async def get_geography(aggregation_unit):
+    request.socket.send_json(
+        {
+            "request_id": request.request_id,
+            "action": "get_geography",
+            "params": {"aggregation_unit": aggregation_unit},
+        }
+    )
+    #  Get the reply.
+    message = await request.socket.recv_json()
+    current_app.logger.debug(f"Got message: {message}")
+    if message["status"] == "done":
+        results_streamer = stream_with_context(assemble_geojson_feature_collection)(
+            message["sql"], message["crs"]
+        )
+        mimetype = "application/geo+json"
+
+        current_app.logger.debug("Returning.")
```

Comment:
Maybe make this message slightly more informative (e.g. to say what we're returning)?